internal/processor/verify_user.go

```diff
@@ -63,9 +47,9 @@ func (p *processor) HandleVerifyUserAction(msg data.ModulePayload) (string, erro
 		return data.FAILURE, err
 	}
 
-	user, err := p.getUserFromAPI(msg.SlackId)
+	user, err := p.usersQ.FilterByUsername(msg.Username).Get()
```

Comment:
I think that it's better to check user existence in module (slack) and in our service (database), otherwise we can verify person that is not in our Slack (e.g. person was deleted in Slack, but not indexed in module yet)